tests/testthat/test_base_benchmark.R

```diff
@@ -271,6 +271,41 @@ test_that("drop option works for BenchmarkResults_operators", {
     wrapper.class = "cl")
 })
 
+test_that("benchmark works with ensemble filters", {
+  lrn = makeLearner("classif.ksvm")
+  lrn = makeFilterWrapper(lrn, fw.method = "E-Borda")
+
+  par.set <- makeParamSet(
+    makeNumericParam("C", lower = -15, upper = 15,
+      trafo = function(x) 2^x),
+    makeNumericParam("sigma", lower = -15, upper = 15,
+      trafo = function(x) 2^x),
+    makeNumericParam("fw.perc", lower = 0, upper = 1),
+    makeDiscreteVectorParam("fw.base.methods", len = 2,
+      values = c("anova.test", "cforest.importance", "FSelector_chi.squared",
+        "FSelector_gain.ratio", "FSelector_information.gain", "FSelector_oneR",
+        "FSelector_relief", "FSelector_symmetrical.uncertainty", "FSelectorRcpp_gain.ratio",
+        "FSelectorRcpp_information.gain", "FSelectorRcpp_symmetrical.uncertainty",
+        "kruskal.test", "praznik_CMIM", "praznik_DISR", "praznik_JMI",
+        "praznik_JMIM", "praznik_MIM", "praznik_MRMR", "praznik_NJMIM",
+        "randomForest.importance", "randomForestSRC.rfsrc", "randomForestSRC.var.select",
+        "ranger.impurity", "ranger.permutation", "variance"))
+  )
+
+  task.names = c("binary", "multiclass")
+  tasks = list(binaryclass.task, multiclass.task)
+  rin = makeResampleDesc("CV", iters = 2L)
+  tune_ctrl = makeTuneControlRandom(maxit = 3)
+
+  tune_wrapper_svm = makeTuneWrapper(lrn, resampling = rin, par.set = par.set,
+    control = tune_ctrl, show.info = FALSE,
+    measures = list(acc))
+
+  expect_class(benchmark(learners = tune_wrapper_svm, task = tasks,
+    resampling = rin, measures = list(acc)), "BenchmarkResult"
+  )
+
```

Comment:
Could you add a test that runs each of the available methods at least once please?

Reply:
I don't think that this is needed here. The purpose is not to test the simple filter execution but the aggregation when being run in combination with an ensemble filter. 
As long as the dedicated simple filter tests are fine there is no need to test over all here again. For simplicity we could even select only two distinct ones to make sure the `makeTuneWrapper()` implementation is able to deal fine with a FilterWrapper containing ensemble filters.

I did not change anything how simple filters are calculated per se. After they have been, the follow up processing is the same for all of them.

Reply:
Ok, I see your point.